pkg/datapath/loader/netlink.go

```diff
@@ -170,7 +170,11 @@ func attachProgram(link netlink.Link, prog *ebpf.Program, progName string, qdisc
 		// Omitting XDP_FLAGS_UPDATE_IF_NOEXIST equals running 'ip' with -force,
 		// and will clobber any existing XDP attachment to the interface.
 		if err := netlink.LinkSetXdpFdWithFlags(link, prog.FD(), int(xdpFlags)); err != nil {
-			return fmt.Errorf("attaching XDP program to interface %s: %w", link.Attrs().Name, err)
+			if option.Config.NodePortAcceleration == option.XDPModeBestEffort {
+				log.Infof("attaching XDP program to interface %s failed. Ignoring", link.Attrs().Name)
```

Comment:
Should we include the error in the log message to display a reason for the failure for debugging purposes?